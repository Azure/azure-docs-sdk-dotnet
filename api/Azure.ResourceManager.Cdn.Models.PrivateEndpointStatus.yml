### YamlMime:NetType
type: struct
implementsWithMoniker:
- value: <xref href="System.IEquatable`1?alt=System.IEquatable&text=IEquatable" data-throw-if-not-resolved="True"/>&lt;<xref href="Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus?alt=Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus&text=PrivateEndpointStatus" data-throw-if-not-resolved="True"/>&gt;
inheritancesWithMoniker:
- values:
  - <xref href="System.ValueType?alt=System.ValueType&text=ValueType" data-throw-if-not-resolved="True"/>
constructors:
- uid: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.#ctor(System.String)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
operators:
- uid: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.op_Equality(Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus,Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.op_Implicit(System.String)~Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.op_Inequality(Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus,Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
methods:
- uid: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.Equals(Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.ToString
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
properties:
- uid: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.Approved
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.Disconnected
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.Pending
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.Rejected
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.Timeout
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus
commentId: T:Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus
namespace: Azure.ResourceManager.Cdn.Models
name: PrivateEndpointStatus
fullName: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus
nameWithType: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus
assembliesWithMoniker:
- value: Azure.ResourceManager.Cdn.dll
packagesWithMoniker:
- value: Azure.ResourceManager.Cdn v1.1.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.Cdn v1.2.0-beta.1
  monikers:
  - azure-dotnet-preview
syntaxWithMoniker:
- lang: csharp
  values:
  - value: 'public readonly struct PrivateEndpointStatus : IEquatable<Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus>'
- lang: fsharp
  values:
  - value: type PrivateEndpointStatus = struct
- lang: vb
  values:
  - value: >-
      Public Structure PrivateEndpointStatus

      Implements IEquatable(Of PrivateEndpointStatus)
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
summary: The approval status for the connection to the Private Link.
metadata:
  api_name:
  - Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus
  api_location:
  - Azure.ResourceManager.Cdn.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus
  - Azure::ResourceManager::Cdn::Models::PrivateEndpointStatus
  - PrivateEndpointStatus
  helpviewer_keywords:
  - Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus struct [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
