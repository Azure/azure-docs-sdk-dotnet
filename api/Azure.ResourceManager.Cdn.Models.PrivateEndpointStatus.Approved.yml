### YamlMime:NetMember
type: property
members:
- returnsWithMoniker:
    type:
    - value: <xref href="Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus?alt=Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus&text=PrivateEndpointStatus" data-throw-if-not-resolved="True"/>
  type: property
  uid: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.Approved
  commentId: P:Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.Approved
  name: Approved
  fullName: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.Approved
  nameWithType: PrivateEndpointStatus.Approved
  syntaxWithMoniker:
  - lang: csharp
    values:
    - value: public static Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus Approved { get; }
  - lang: fsharp
    values:
    - value: 'static member Approved : Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus'
  - lang: vb
    values:
    - value: Public Shared ReadOnly Property Approved As PrivateEndpointStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
  summary: Approved.
  metadata: {}
uid: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.Approved*
commentId: Overload:Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.Approved
namespace: Azure.ResourceManager.Cdn.Models
name: Approved
fullName: Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.Approved
nameWithType: PrivateEndpointStatus.Approved
assembliesWithMoniker:
- value: Azure.ResourceManager.Cdn.dll
packagesWithMoniker:
- value: Azure.ResourceManager.Cdn v1.1.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.Cdn v1.2.0-beta.1
  monikers:
  - azure-dotnet-preview
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  api_name:
  - Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.Approved
  - Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.get_Approved
  api_location:
  - Azure.ResourceManager.Cdn.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.Cdn.Models.PrivateEndpointStatus.Approved
  - Azure::ResourceManager::Cdn::Models::PrivateEndpointStatus::Approved
  - Approved
  - PrivateEndpointStatus.Approved
  - PrivateEndpointStatus::Approved
  helpviewer_keywords:
  - PrivateEndpointStatus.Approved property [.NET]
  - Approved property [.NET], struct PrivateEndpointStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
